pkg/apis/prow/prow.go

```diff
@@ -67,7 +71,7 @@ type ProwJobSpec struct {
 }
 
 type ProwJobStatus struct {
-	StartTime        time.Time               `json:"startTime,omitempty"`
+	StartTime        bigquery.NullTimestamp  `json:"startTime,omitempty"`
```

Comment:
I don't think you want to change this.  Keep it as time.Time

Check IsValid for bigqueryProwJobRun StartTime before it gets converted to ProwJobStatus and if it isn't valid log an error and skip creating that job, go on to the next one.

 https://github.com/openshift/sippy/blob/c32e1fcf0bfab7b2c7254a468509d6c1aa464c8a/pkg/dataloader/prowloader/bigqueryjobs.go#L99

Reply:
nice suggestion Thanks.